chromium_src/chrome/install_static/install_modes.cc

```diff
@@ -1,35 +1,72 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
I think we don't need to touch license of existing file?

Reply:
Sorry, not need to touch this, yes.